Qfront/src/redux/api.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+// Create base url for axios to
+const axiosInstance = axios.create({
+    baseURL: process.env.REACT_APP_API_URL,
+});
+
+export const baseUrl = axiosInstance;
```

Comment:
`baseUrl` is not a descriptive name for the axios instance. `axiosInstance` is much better.